js/script.js

```diff
@@ -20,13 +20,83 @@ function Generator() {
 
 }
 
+function getRapName(name, first_names, last_names) {
```

Comment:
it feels like these 'generation' methods should probably be behaviors of Generator.